modules/hotmaps/1.0/config/default.yaml

```diff
@@ -0,0 +1,92 @@
+lcr-modules:
+    
+    hotmaps:
+
+        inputs:
+            # Available wildcards: {seq_type}
+            # No genome_build wildcard because only grch37 is compatible!
+            # Make sure input MAF paths are for grch37 only
+            input_maf: "__UPDATE__"
+            subsetting_categories: "__UPDATE__"
+
+        maf_processing:
+            sample_sets: ["__UPDATE__"]
+            prepare_mafs: "{SCRIPTSDIR}/generate_smg_inputs/1.0/generate_smg_inputs.R"
+            include_non_coding: False
+            blacklists: []
+            blacklist_drop_threshold: 4
+        
+        vcf2maf:
+            # Path pointing to VEP files for running vcf2maf e.g "ref/ensembl_vep_cache"
+            vep_cache: "__UPDATE__"
+
+        # Path to HotMAPS repo containing updated scripts
+        hotmaps_repo: "https://github.com/morinlab/HotMAPS/archive/refs/heads/master.zip"
+
+        pdb_stucture_dirs:
+            # modbase_dir: set to the directory containing the ModBase theoretical protein structures 
+            modbase_dir: "__UPDATE__"
+            # pdb_dir: set to the directory containing PDB structures downloaded from wwPDB database
+            pdb_dir: "__UPDATE__"
+            # refseq_homology_dir: specify the path containing the ModBase RefSeq structures. 
+            # %(modbase_dir)s placeholder will be replaced with path set in modbase_dir above
+            refseq_homology_dir: "%(modbase_dir)s/ModBase_H_sapiens_2013_refseq/models/model/"
+            # ensembl_homology_dir: specify path containing ModBase Ensembl structures
+            ensembl_homology_dir: "%(modbase_dir)s/H_sapiens_2013/models/"
+            # biological_assembly_dir: specify path containing wwPDB biological structures
+            biological_assembly_dir: "%(pdb_dir)s/biounit/coordinates/all/"
+            # non_biological_assembly_dir: specify the path containing wwPDB PDB structures
+            non_biological_assembly_dir: "%(pdb_dir)s/structures/all/pdb/"
+
+        options:
+            mysql:
+                mysql_host: "__UPDATE__"
+                mysql_user: "__UPDATE__"
+                mysql_db: "__UPDATE__"
+                mysql_passwd: "__UPDATE__"
```

Comment:
of course in the comment above I meant giving examples without showing password, this is self-explanatory :smile: